scripts/tasks/perf-test.js

```diff
@@ -90,20 +101,35 @@ async function runScenarioNTimes(page, times) {
     let peritem = await page.$eval('.peritem', result => result.innerText);
 
     // add perf numbers
-    totalsum += parseFloat(total.replace(/[a-zA-Z:]/g, ''));
-    peritemsum += parseFloat(peritem.replace(/[a-zA-Z:]/g, ''));
+    totalsamples.push(parseFloat(total.replace(/[a-zA-Z:]/g, '')));
+    peritemsamples.push(parseFloat(peritem.replace(/[a-zA-Z:]/g, '')));
 
     // reset
     await runTestButton.click();
   }
 
-  // average
   return {
-    total: (totalsum / times).toFixed(3),
-    peritem: (peritemsum / times).toFixed(3)
+    totals: totalsamples,
+    peritem: peritemsamples,
```

Comment:
minor nit, can you make these camelCased for consistency, just hit f2 and fix a couple of these